src/pages/documentation/platform-assurance/tutorials/optimize-configuration-view.md

```diff
@@ -0,0 +1,35 @@
+# Optimize Setup Validation View
+
+## Overview
+
+You may use the Optimize Setup Validation view in order to quickly see if your app has been successfully set up to use AEP Optimize. The view validates all pre-requisites have been met, and offers feedback on how to fix your setup in cases where requirements have been missed.
+
+## Getting Started
+
+To use this view, complete the following steps:
+
+1. [Set up Assurance](../set-up-project-griffon.md)
+2. [Create](./README.md#creating-sessions) and [connect](./README.md#connecting-to-a-session) to a Assurance session
```

Comment:
Same comment here - we need to update these links to the proper links within this repo.

So I think it's `set-up.md` for Set up Assurance, and `index.md` instead of `README.md`.